recipes/mariadb-connector-c/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e -x
+
+mkdir build
+cd build
+
+cmake \
+    -DENABLE_FORMAT=OFF \
+    -DBUILD_SHARED_LIBS=ON \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON \
+    -DWITH_MYSQLCOMPAT=1 \
+    -DINSTALL_LIBDIR=lib \
+    ..
+
+make -k -j${CPU_COUNT} || true
```

Comment:
Why do you have the `|| true`?

Reply:
Im not sure what `|| true` does. Copied it from another feedstock and forgot to test without it.